targets/TARGET_TOSHIBA/TARGET_TMPM46B/device/TOOLCHAIN_ARM_STD/tmpm46bf10fg.sct

```diff
@@ -22,6 +22,14 @@
   #define MBED_APP_SIZE 0x100000
 #endif
 
+#if !defined(MBED_RAM_START)
+#define MBED_RAM_START 0x20000000
+#endif
+
+#if !defined(MBED_RAM_SIZE)
+#define MBED_RAM_SIZE 0x30000
```

Comment:
The RAM size was `0x80000`. See what was there at line 52. Is there a reason this was changed?